### PR TITLE
5 view an individual article

### PR DESCRIPTION
**Pull request by stevelw**

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          title,
+          body,
+          topic: topics[topic] ?? topic,
+          timestamp,
+          article_img_url,
+        });
+        setIsLoading(false);
+      })
+      .catch((err) => setIsError(true));
+  }, []);
```
Comment (ncjaycarroll) [inline src/components/Article.jsx:59]: article will already come back as an object, probably better to just setArticle to what comes back from the api call and access what you need from it, no need to destructure and set each property manually as you do here

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -15,10 +16,12 @@ export default function ArticleList({ articles }) {
   return (
     <div style={style}>
       {articles.map(({ article_id, title, topic, article_img_url }) => {
+        const urlFriendlyTitle = title.replaceAll(/[^a-z]/gi,'-')
+        const articleUrl = '/articles/' + urlFriendlyTitle + '-' + article_id
         return (
           <ArticleCard key={article_id}>
             <div style={childStyle}>
-              <h3>{title}</h3>
+              <Link to ={articleUrl}><h3>{title}</h3></Link>
```
Comment (ncjaycarroll) [inline src/components/ArticleList.jsx:24]: Is this necessary ?
could you just link to {`/articles/article.article_id }

**Review by ncjaycarroll — COMMENTED** [review]
Potentially some overcomplicated code to reconsider refactoring
Some functionality for the back button on the article page would be a good addition
Other than that looking good so far

File: src/components/ArticleList.jsx (hunk 2 above)
Comment (stevelw) [reply src/components/ArticleList.jsx:24]: Not necessary, but makes the URLs human readable for social sharing.

**Pull request merged**